terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -177,7 +177,7 @@ module "server_containerized" {
   }
   main_disk_size = 3000
   runtime = "podman"
-  container_repository = "registry.suse.de/suse/sle-15-sp6/update/products/manager50/containerfile/suse/manager/5.0/x86_64"
+  container_repository = "registry.suse.de/suse/maintenance/35237/suse_sle-15-sp6_update_products_manager50_update_containerfile/suse/manager/5.0/x86_64"
```

Comment:
please add a comment explaining it's temporary, and you can even link my github card to change this in the future - https://github.com/SUSE/spacewalk/issues/25120

Reply:
Done.